src/main/resources/db/migration/V1_74__appointment_session_type.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TYPE appointment_session_type AS ENUM ('ONE_TO_ONE','GROUP');
+
+alter table appointment_delivery add column appointment_session_type appointment_session_type default 'ONE_TO_ONE';
+
+alter table appointment_delivery alter column appointment_session_type drop default;
```

Comment:
_zero downtime deploy behaviour_

🙋‍♂️ As far as I can see, it's possible that
- old service code
- running after this migration executed
- handles a delivery creation

In that case, the old code wouldn't supply the `appointment_session_type` column, and would then result in a SQL error without a default.

It's safer to either
- `drop default` in a subsequent separate deployment or post-rollout migration
- execute creating the new column migration before rolling out this code + `drop default`

Reply:
ok. I've removed the `drop default`.